docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,119 @@
+ - Create 'agent' machines to ping app-servers using [controllers](https://github.com/mattermost/mattermost-load-test-ng/tree/61c44f35224b76d3098199b0cd2b67db2222b549/loadtest/control)
+ - Populate database, and start app, agents (i.e loadtest)
+ - Collect metrics from app, and agent deployments to either,
+    - manage an unbounded load test
+    - analyse api performance after the load-test completes.
+
+Loadtest instances created with this framework achieve the same goals as mentioned above, only some of the things like creating a deployment, running a loadtest, etc. are automated.
+
+Following are some steps to load-test a new feature in production, after testing new actions locally.
+
+ - Go through [terraform_loadtest.md](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md)
+ - Some additional information on the above document
+    - AWS credentials are to be fetched from [onelogin](https://mattermost.onelogin.com/) and [added to terraform config](https://registry.terraform.io/providers/hashicorp/aws/latest/docs). <!-- docfix -->
+    - Enterprise license is to be fetched from [Developers:private](https://community.mattermost.com/private-core/channels/core-developers) <!-- docfix;  https://github.com/mattermost/enterprise/pull/1208 -->
+    - When performing [this](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#copy-and-modify-the-required-configuration) step, edit `MattermostLicenseFile` value to the path containing the license.
+        - The fields `MattermostDownloadURL` and `LoadTestDownloadURL` point to the latest mattermost-server, and load-test package respectively to be used in the load-test.
+        - That's the default option, when there's unmerged changes to `mattermost-server`, or `mattermost-load-test-ng`
+            - `make build-linux` in `mattermost-server` directory, change `MattermostDownloadURL` value to the path containing mattermost executable. For example `file:///somepath/mattermost-server/bin/linux_amd64/mattermost`
+            - `make package` in `mattermost-load-test-ng` directory, change `LoadTestDownloadURL` value to the path containing gzip of load-test package. For example `file:///somepath/mattermost-load-test-ng/dist/v1.5.0-8-gd4f18cf/mattermost-load-test-ng-v1.5.0-8-gd4f18cf-linux-amd64.tar.gz`
+    - Edit `SSHPublicKey` in deployer.json after setting up ssh.
+    - `go run ./cmd/ltctl deployment create`
+        <!-- devfix; anything to do with deployment spawns zombie processes if the command is cancelled with a shell interruption. Further ltctl deployment commands don't work, so one has to restart the computer before starting again. Haven't tried to kill the processes manually.-->
+        - Limit operations of `deployment` to a single shell window.
+        - If the deployment gets stuck, check for `ps -ef | grep terraform`, if there are running processes, restart the computer and start again.
```

Comment:
I am not sure why you would need to restart the computer. If the deployment gets stuck, we can try to run the same command again, or destroy it.

Reply:
Hi @agnivade , it was this thread: https://community.mattermost.com/core/pl/u49ya4pybby9tyde54qxw198ey

I've updated instructions accordingly.